### PR TITLE
Fix file path issue in check_fonts_installed method

### PR DESCRIPTION
**Pull request by irfano**
## What does it do?
This fixes an issue with accessing a file in `check_fonts_installed` method. Before, calling this from WCAndroid resulted in the following error:
```
No such file or directory @ rb_sysopen - playstoreres/assets/style.css
```

## Checklist before requesting a review

- [ ] Run `bundle exec rubocop` to test for code style violations and recommendations
- [ ] Add Unit Tests (aka `specs/*_spec.rb`) if applicable
- [ ] Run `bundle exec rspec` to run the whole test suite and ensure all your tests pass
- [x] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the appropriate existing `###` subsection of the existing `## Trunk` section.
- [ ] If applicable, add an entry in [the `MIGRATION.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/MIGRATION.md) to describe how the changes will affect the migration from the previous major version and what the clients will need to change and consider.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please add an entry in the CHANGELOG.md file to describe the changes made by this PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">No reviewers have been set for this PR yet. Please request a review from <strong>@⁠wordpress-mobile/apps-infrastructure</strong>.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb [hunk 1]
```diff
@@ -63,7 +63,8 @@ def check_fonts_installed!(config:)
         # Parse the first font in each `font-family` attribute found in all found CSS files.
         # Only the first in each `font-family` font list matters, as others are fallbacks we don't want to use anyway.
         font_families = all_stylesheets.flat_map do |s|
-          File.readlines(s).flat_map do |line|
+          stylesheet_path = resolve_path(s) if can_resolve_path(s)
+          File.readlines(stylesheet_path).flat_map do |line|
```
Comment (AliSoftware) [inline lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb]: If `can_resolve_path(s)` returns `false`, that means that `stylesheet_path` will be `nil`, and `File.readlines(nil)` will crash with an unclear error.

The `resolve_path` method already handles crashing with a properly-handled `UI.crash!` that prints the path it failed to resolve if that's the case, which would make a better error than having `File.readlines(nil)` crash with an error not mentioning the path that caused the issue. So I think we can just remove the `if can_resolve_path(s)` here to handle those error cases gracefully already?

```suggestion
          stylesheet_path = resolve_path(s)
          File.readlines(stylesheet_path).flat_map do |line|
```

**Review by AliSoftware — CHANGES_REQUESTED** [review]

File: lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb (hunk 1 above)
Comment (irfano) [reply lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb]: Good suggestion. I could see "[!] Unable to locate assets/style.css" error message after applying your suggestion. Thank you!

**Review by AliSoftware — APPROVED** [review]

**Comment by irfano** [comment]
@AliSoftware, could you help me choose the right milestone for this PR and #614? I want to include these fixes in the next WCAndroid and WCiOS releases.

**Comment by AliSoftware** [comment]
Cc @iangmaia since I'm kinda afk today

- I think we could remove the Danger rule about milestone in this repo as we don't really use them for this tooling repo
- Can you make a new release so that @irfano can point to it in WC repos?

**Comment by AliSoftware** [comment]
I've prepared a new release in https://github.com/wordpress-mobile/release-toolkit/pull/616

**Comment by AliSoftware** [comment]
@irfano Version `12.3.3` has now been released and published to RubyGems, so you should be able to update the reference in WC repos 👍

**Pull request merged**